CHANGELOG.md

```diff
@@ -4,6 +4,75 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.5.0 - 2018-12-03
+
+### Added
+
+- We now push the ingress custom resource defintion and initial
+  configuration, allowing the ingress operator to configure itself
+  without referencing the deprecated `cluster-config-v1` resource.
+
+### Changed
+
+- Pull secret documentation now points to
+  [try.openshift.com](https://try.openshift.com) for pull-secret
+  acquisition, instead of pointing at `account.coreos.com`.  Users
+  will need to update their pull secrets.
+- If the automatic bootstrap teardown (which landed in 0.4.0) times
+  out waiting for the `bootstrap-complete` event, the installer exits
+  with a non-zero exit code.  We had ignored watcher timeouts in 0.4.0
+  due to concerns about watcher robustness, but the current watcher
+  code has been reliable in our continuous integration testing.
+- The hard-coded `quay.io/coreos/bootkube` dependency has been
+  replaced by the new [cluster-bootstrap][] image, which is referenced
```

Comment:
Empty link.

Reply:
And did that actually land?

Reply:
> Empty link.

It's using "[the implicit link name shortcut][1]" ;).

> And did that actually land?

Yes, #760.

[1]: https://daringfireball.net/projects/markdown/syntax#link

Reply:
Huh. I've always just omitted the trailing `[]`. Either works though, according to CommonMark (everyone should be using this at this point) and Markdown/GFM.